.gitignore

```diff
@@ -24,13 +24,12 @@
 !\.zuul
 !\.zuul.yaml
 
-/bazel-*
-/mobile/bazel-*
+bazel-*
+MODULE.bazel.lock
```

Comment:
im still unconvinced about the wisdom of not checking in lock files

Reply:
yea i think once we're on 7.4+ we can check them in, just have to add a CI job validating they're up to date. the format isn't very stable in 6.x tho